R/bound_prediction.R

```diff
@@ -0,0 +1,38 @@
+#' Truncate a numeric prediction column
+#'
+#' For user-defined lower_limit and/or upper_limit bound, ensure that the values in the
+#' `.pred` column are coerced to these bounds.
+#'
+#' @param x A data frame that contains a numeric column named `.pred`.
+#' @param lower_limit,upper_limit Single numerics (or `NA`) that define
+#' constrains on `.pred`.
+#' @param call The call to be displayed in warnings or errors.
+#' @return `x` with potentially adjusted values.
+#' @examples
+#' data(solubility_test, package = "yardstick")
+#'
+#' names(solubility_test) <- c("solubility", ".pred")
+#'
+#' bound_prediction(solubility_test, lower_limit = -1)
+#' @export
+bound_prediction <- function(x, lower_limit = -Inf, upper_limit = Inf,
+                             call = rlang::caller_env()) {
```

Comment:
```suggestion
                             call = rlang::current_env()) {
```

The calling environment from where this is evaluated is likely one step up from where we want it. If you'd like to call `bound_predictions()` yourself and see `Error in bound_predictions()`, we'll need to bump this down one frame. 

This change will affect snaps. :)